deployment/conf/supported_apps_w_extensions.json

```diff
@@ -117,7 +117,7 @@
       "xls",
       "xlsx"
     ],
-    "upload_specification": [
+    "import_specification": [
```

Comment:
The upload to import change makes sense aligning to documentation

Reply:
I think import specification works fine as a name. I can't think of anything that's head-and-shoulders better. "Import mapping" or "bulk import manifest" possibly, but I don't have strong feelings about either of those. @egdow thoughts?

Reply:
Manifest seems like more of a csv directory term. I think specification works for this purpose. My usual thoughts are to keep the language the same across importing as much as possible